packages/fgtclb/academic-projects/Resources/Private/Partials/Project/DemandActiveState.html

```diff
@@ -0,0 +1,40 @@
+<html
+    xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
+    xmlns:ace="http://typo3.org/ns/FGTCLB/AcademicProjects/ViewHelpers"
+    data-namespace-typo3-fluid="true"
+>
+    <f:variable name="activeStateOptions" value="{0:'all', 1:'active', 2:'completed'}" />
+    <label for="activeState">
+        {f:translate(
+            key: 'activeState.label',
+            extensionName: 'academic_projects'
+        )}
+    </label>
+    <f:form.select
+        id="activeState"
+        property="activeState"
+        value="{demand.activeState}"
+        additionalAttributes="{'onchange': 'this.form.submit()'}"
```

Comment:
inline js is evil. I get the background that a "workable" state should be shipped,
but we need to make that configurable in some way in the future.

Should also be evaluated if this clashes with content security policies in some way,
not a experienced person here. Needs to be evaluated and tested ! (Frontend CSP).